.github/workflows/publish.yml

```diff
@@ -53,3 +53,33 @@ jobs:
           push: true
           tags: ${{ steps.docker_metadata.outputs.tags }}
           labels: ${{ steps.docker_metadata.outputs.labels }}
+
+  dockerhub-description:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Find change in docker readme file
+        id: docker-readme-change
+        uses: tj-actions/changed-files@v24
+        with:
+          files: |
+            docker-readme.md
+
+      - name: add changelog
+        if: steps.docker-readme-change.outputs.any_changed == 'true'
+        run: |
+          cat CHANGELOG.md >> docker-readme.md
```

Comment:
I think this is too much for a docker image description, the repository will be referenced in the description so the changelog will be findable.

Reply:
yes sorry I forgot to remove this. It's should be all right now.